compatibility/common/src/main/java/org/projectnessie/tools/compatibility/internal/CurrentNessieServer.java

```diff
@@ -54,7 +54,7 @@ static CurrentNessieServer currentNessieServer(
 
   @Override
   public URI getUri() {
-    return jersey.getUri();
+    return jersey.getUri().resolve("v1");
```

Comment:
Don't we need something similar (in the future?) for the old servers? And have a "mapping" which Nessie versions support with API versions?

Reply:
Yeah, I have not thought in details about the older clients / old servers tests yet, but I suppose we'd use `v1` until the switch over version, then `v2` (`v1` tests in current servers always run during normal CI). Older clients would always use `v1` when talking even to new servers.